quotaservice-cli/client/client.go

```diff
@@ -199,22 +192,21 @@ func connectToServer(method, url string, data ...[]byte) *http.Response {
 		panic("Should never get here")
 	}
 
-	r, e := http.NewRequest(method, url, dataReader)
-	kingpin.FatalIfError(e, "HTTP error")
+	cmdTokens := []string{"-sS", "-X", method, url, "-H", "Content-Type: application/json"}
+	if dataReader != nil {
+		cmdTokens = append(cmdTokens, "--data", string(data[0]))
+	}
 
-	client := &http.Client{}
-	resp, e := client.Do(r)
-	kingpin.FatalIfError(e, "HTTP error")
+	cmd := exec.Command("beyond-curl", cmdTokens...)
```

Comment:
I'm fine with this change, but FYI there's github.com/squareup/beyondclient-go

Reply:
I'll take a look. But I'm inclined to merge this PR for now to unblock a user.

I'm not sure if that is going to introduce a circular dependency, as this repo is used by the monorepo. Maybe not, as it's not directly imported in the code, I'll check later.